packages/app/src/templates/services/template.service.test.ts

```diff
@@ -115,7 +118,7 @@ describe("`TemplateService`", () => {
   describe("`updateTemplateOrder` method", () => {
     test("map data and returns success when `deviceService.request` returns success", async () => {
       deviceService.request = jest.fn().mockReturnValue(successResponse)
-      const response = await subject.updateTemplateOrder(template)
+      const response = await subject.updateTemplatesOrder([template])
```

Comment:
Maybe it would be good to add another template to the array? By that we could test e.g. how many times `deviceService.request` was called and if proper params are passed to the function for every call.

Reply:
Added in [43e06d1](https://github.com/mudita/mudita-center/pull/804/commits/43e06d134aad31f5e351a09965e5574fbf64604f)